sdk/search/Azure.Search.Documents/tests/Serialization/SearchFilterTests.cs

```diff
@@ -178,30 +174,29 @@ public void Polygons()
                 "geo.intersects(Foo, geography'POLYGON((0 0,0 1,1 1,0 0))')",
                 SearchFilter.Create($"geo.intersects(Foo, {line})"));
 
-            GeoPolygon polygon = new GeoPolygon(new[] { line });
-            Assert.AreEqual(
-                "geo.intersects(Foo, geography'POLYGON((0 0,0 1,1 1,0 0))')",
-                SearchFilter.Create($"geo.intersects(Foo, {polygon})"));
+            //GeoPolygon polygon = new GeoPolygon(new[] { line });
+            //Assert.AreEqual(
+            //    "geo.intersects(Foo, geography'POLYGON((0 0,0 1,1 1,0 0))')",
+            //    SearchFilter.Create($"geo.intersects(Foo, {polygon})"));
```

Comment:
Nit: could just clean up the code to remove these since it's all in source control.